### PR TITLE
Add pykickstart.spec.in for packit

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This pull request is to follow up #259 .

The upstream repository contains temporary `pykickstart.spec.in` the file which is generated by Makefile `make pykickstart.spec`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -109,6 +109,10 @@ bumpver: docs
 	make -C po pykickstart.pot ; \
 	zanata push $(ZANATA_PUSH_ARGS)
 
+pykickstart.spec:
+	cp pykickstart.spec.in pykickstart.spec
+	sed -i "s/%%VERSION%%/$(VERSION)/" pykickstart.spec
```
Comment (dcantrell) [inline Makefile]: Style concern here.  Can you do this in one line instead, like:

sed -e "s/%%VERSION%%/$(VERSION)/" < $< > $@

Changing the target so it lists pykickstart.spec.in as a requirement

File: Makefile [hunk 2]
```diff
@@ -109,6 +109,10 @@ bumpver: docs
 	make -C po pykickstart.pot ; \
 	zanata push $(ZANATA_PUSH_ARGS)
 
+pykickstart.spec:
```
Comment (dcantrell) [inline Makefile]: Change to:

pykickstart.spec: pykickstart.spec.in

File: Makefile (hunk 1 above)
Comment (phracek) [reply Makefile]: This can be solved also by the command `sed "s/%%VERSION%%/$(VERSION)/" $< > $@`.
But I will use your approach.

**Comment by dcantrell** [comment]
Thanks for the update.  I'm going to merge this now, but there's still a bit I want to do for the spec file before it's ready to generate packages for submission via packit.

**Pull request merged**